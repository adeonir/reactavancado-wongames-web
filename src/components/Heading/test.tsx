import { render, screen } from '@testing-library/react'

import { Heading } from '.'

describe('<Heading />', () => {
  it('should render the heading', () => {
    const {
      container: { firstChild },
    } = render(<Heading>Most Populars</Heading>)

    expect(
      screen.getByRole('heading', { name: /most populars/i }),
    ).toBeInTheDocument()

    expect(firstChild).toMatchSnapshot()
  })
})
