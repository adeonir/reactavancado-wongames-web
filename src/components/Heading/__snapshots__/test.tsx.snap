// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> should render the heading 1`] = `
<h2
  class=""
>
  Most Populars
</h2>
`
